docs/api-docs/provider-alibaba.md

```diff
@@ -1,7 +1,5 @@
 ---
 title: provider-alibaba
```

Comment:
Just double checking, these provider docs will show in TOC right?

Reply:
Yes, that's why this looks like a weird implementation. The pages have to be here with this metadata to appear in the menu but we can't enforce a redirect from the markdown page so there are a few hoops that have to be jumped through to make it happen. 

You can see in the preview site for the crossplane.github.io PR. https://master.d1bmjmu5ixie8d.amplifyapp.com/v1.9/api-docs/